### PR TITLE
Update the wording and screenshots for GH projects

### LINKED ISSUE #237: Check the episodes with screenshots of PyCharm, GitHub Milestones and projects and check they are up-to-date

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #237.

### CODE REVIEW

File: _episodes/51-managing-software.md [hunk 1]
```diff
@@ -308,32 +315,31 @@ and automatically moved between columns based on their progress or status.
 > to help you plan and track your team's work effectively.
 {: .callout}
 
-Let's create a Project in GitHub to plan the first release of our code.
+Let's have a quick look at how Projects are created in GitHub - we will not use them much in 
+this course but it is good to be aware of make use of them when suitable.
```
Comment (steve-crouch) [inline _episodes/51-managing-software.md:319]:
```suggestion
this course but it is good to be aware of how to make use of them when suitable.
```

File: _episodes/51-managing-software.md [hunk 2]
```diff
@@ -299,7 +299,14 @@ such as issues and pull requests -
...
 
-> ## Project are a Cross-Repository Management Tool
+GitHub projects are "an adaptable, flexible tool for planning and tracking work on GitHub" -
+they now provide interchangeable spreadsheet, task-board, or roadmap views of your project 
+that integrates with your issues and pull requests on GitHub to help you 
+plan and track your work effectively.
+We recommend you to have a look at [GitHub’s documentation on Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) 
+and see if they are suitable for your software development workflow.
+
+> ## GitHub Project are a Cross-Repository Management Tool
```
Comment (steve-crouch) [inline _episodes/51-managing-software.md:309]:
```suggestion
> ## GitHub Projects are a Cross-Repository Management Tool
```

File: _episodes/51-managing-software.md [hunk 3]
```diff
@@ -308,32 +315,31 @@ and automatically moved between columns based on their progress or status.
...
+types/views that you customise yourself: 
+      - Table - a spreadsheet-style table to filter, sort and group your issues and pull requests. 
+      - Board - a "cards on a board" view of the project, with issues and pull requests being 
+      spread across customizable columns as cards on kanban board
+      - Roadmap - suitable for a high-level visualisation of your project over time. 
+![Selecting a project board template in GitHub](../fig/github-project-template.png){: .image-with-shadow width="800px"}
+  Regardless of which project type/view you select, you can easily switch to a different 
+  project layout later on.
+
+3. For example, select the "Board" type for the project and fill in the name of your project -
```
Comment (steve-crouch) [inline _episodes/51-managing-software.md:337]:
```suggestion
3. For example, select the "Board" type for the project, fill in the name of your project
```

File: _episodes/51-managing-software.md [hunk 4]
```diff
@@ -308,32 +315,31 @@ and automatically moved between columns based on their progress or status.
...
+      - Table - a spreadsheet-style table to filter, sort and group your issues and pull requests. 
+      - Board - a "cards on a board" view of the project, with issues and pull requests being 
+      spread across customizable columns as cards on kanban board
+      - Roadmap - suitable for a high-level visualisation of your project over time. 
+![Selecting a project board template in GitHub](../fig/github-project-template.png){: .image-with-shadow width="800px"}
+  Regardless of which project type/view you select, you can easily switch to a different 
+  project layout later on.
+
+3. For example, select the "Board" type for the project and fill in the name of your project - 
+e.g. "Inflammation project - release v0.1".
```
Comment (steve-crouch) [inline _episodes/51-managing-software.md:338]:
```suggestion
(e.g. "Inflammation project - release v0.1"), and select `Create project`.
```

File: _episodes/51-managing-software.md [hunk 5]
```diff
@@ -308,32 +315,31 @@ and automatically moved between columns based on their progress or status.
...
+4. After it is created, you should also populate the description of the project from the project's Settings,
+   which can be found by clicking the `...` button in the top right corner of the project.
+![Project board setting in GitHub](../fig/github-project-settings.png){: .image-with-shadow width="800px"}
+![Adding project description and metadata in GitHub](../fig/github-project-description.png){: .image-with-shadow width="800px"}
```
Comment (steve-crouch) [inline _episodes/51-managing-software.md:342]:
```suggestion
![Adding project description and metadata in GitHub](../fig/github-project-description.png){: .image-with-shadow width="800px"}
   After adding a description, select `Save`.
```

**Review by steve-crouch — CHANGES_REQUESTED** [review]
Looks good, only some minor suggestions and corrections.

**Review by anenadic — COMMENTED** [review]
All comments make sense and accepted - thank you @steve-crouch !

**Review by steve-crouch — APPROVED** [review]

**Pull request merged**